borgcron_settings.sh

```diff
@@ -24,6 +25,4 @@ ADD_BACKUP_PARAMS="" # --one-file-system for backing up root file dir
 PRUNE_PARAMS="--keep-daily=14 --keep-weekly=8 --keep-monthly=6 --keep-yearly=0"
 # for web servers (only disaster recovery:) --keep-daily=7 --keep-weekly=4 --keep-monthly=1 --keep-yearly=0
 
-# include main script
 # shellcheck disable=SC1091
```

Comment:
You can remove this shellcheck comment too now. It was only needed for sourcing the main script below.